CHANGES/10154.bugfix.rst

```diff
@@ -0,0 +1 @@
+Updated :meth:`aiohttp.web.StreamResponse.write` annotation to also allow ``bytearray`` and ``memoryview`` as inputs -- by :user:`cdce8p`.
```

Comment:
I think if you do a single backtick for `bytearray` and `memoryview` it will link to the standard lib docs for them

Reply:
Didn't work. Not sure why, maybe because these are builtins?
I'd suggest we leave it as is.

Reply:
maybe :bulitin:`bytearray` or :class:`bytearray` might work